eox_nelp/utils.py

```diff
@@ -142,3 +142,15 @@ def get_item_label(item):
         return matches.group(1)
 
     return ""
+
+
+def camel_to_snake(string):
```

Comment:
Maybe this util could have test, but anyway is not mandatory.

Reply:
Yes, this should have its own test